osu.Game/Screens/Multi/Lounge/Components/DrawableRoom.cs

```diff
@@ -36,6 +38,8 @@ public class DrawableRoom : OsuClickableContainer, IStateful<SelectionState>, IF
 
         public event Action<SelectionState> StateChanged;
 
+        public Action DuplicateRoom;
```

Comment:
Not sure about having this as a publically settable action and setting it outwardly rather than locally DI'ing whatever is necessary here and setting the action within. Is this ever going to have a different implementation?

Also, all references to "duplicating rooms" in code should be also renamed to `CreateCopy` or similar if we're renaming the action on the UI.